doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -30,7 +30,12 @@ Notations
 Basic notations
 ~~~~~~~~~~~~~~~
 
-.. cmd:: Notation @string := @one_term {? ( {+, @syntax_modifier } ) } {? : @scope_name }
+.. cmd:: Notation @notation_declaration
```

Comment:
IIUC the issues/questions for me have been resolved?

Optional: I'd use the shorter name "notation_decl" for the sake of readability and layout.  A shortcut for this is to add `| notation_declaration notation_decl` in `common.edit_mlg` after the line `| bindings_with_parameters alias_definition` and then adjust the .rst file as needed.

Reply:
> The issues/questions for me have been resolved?

Yes